yogstation/code/game/objects/items/tools.dm

```diff
@@ -2,67 +2,70 @@
 //the new and improved jaws
 /obj/item/jawsoflife
 	name = "jaws of life"
-	
+	desc = "A set of jaws of life, compressed through the magic of science. It's fitted with a prying head."
 	materials = list(MAT_METAL=150,MAT_SILVER=50,MAT_TITANIUM=25)
-	desc = "A set of jaws of life, compressed through the magic of science. It's fitted with a cutting head."
 	icon = 'icons/obj/tools.dmi'
-	icon_state = "jaws_cutter"
+	icon_state = "jaws_pry"
 	item_state = "jawsoflife"
 	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
 	w_class = WEIGHT_CLASS_SMALL
-
-	usesound = 'sound/items/jaws_cut.ogg'
+	usesound = 'sound/items/jaws_pry.ogg'
 	force = 15
 	toolspeed = 0.7
+	tool_behaviour = TOOL_CROWBAR
+
+/obj/item/jawsoflife/proc/transform_crowbar(mob/user)
+	desc = "A set of jaws of life, compressed through the magic of science. It's fitted with a prying head."
+	attack_verb = list("attacked", "bashed", "battered", "bludgeoned", "whacked")
+	usesound = 'sound/items/jaws_pry.ogg'
+	hitsound = 'sound/items/jaws_pry.ogg'
+	tool_behaviour = TOOL_CROWBAR
+	icon_state = "jaws_pry"
+	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
+	if (iscyborg(user))
+		to_chat(user,"<span class='notice'>Your servos whirr as the cutting head reconfigures into a prying head.</span>")
+	else
+		to_chat(user, "<span class='notice'>You attach the pry jaws to [src].</span>")
+	update_icon()
+
+/obj/item/jawsoflife/proc/transform_cutters(mob/user)
+	attack_verb = list("pinched", "nipped")
+	icon_state = "jaws_cutter"
+	hitsound = 'sound/items/jaws_cut.ogg'
+	usesound = 'sound/items/jaws_cut.ogg'
 	tool_behaviour = TOOL_WIRECUTTER
+	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
+	desc = "A set of jaws of life, compressed through the magic of science. It's fitted with a cutting head."
+	if (iscyborg(user))
+		to_chat(user,"<span class='notice'>Your servos whirr as the prying head reconfigures into a cutting head.</span>")
+	else
+		to_chat(user, "<span class='notice'>You attach the cutting jaws to [src].</span>")
+	update_icon()
 
 //jaws of life suicide code
 /obj/item/jawsoflife/suicide_act(mob/user)
-	switch(tool_behaviour)
```

Comment:
oop here

Reply:
Thank you!
I was pretty tired when I made that change so its nice to have someone point out the obvious that I missed.